tests/transforms/test_defer_measurements.py

```diff
@@ -995,8 +1051,33 @@ def qnode2(parameters):
             assert np.allclose(op1.data, op2.data)
 
 
-class TestQubitReset:
-    """Tests for the qubit reset functionality of `qml.measure`."""
+class TestQubitReuseAndReset:
+    """Tests for the qubit reuse/reset functionality of `qml.measure`."""
+
+    def test_new_wire_for_multiple_measurements(self):
+        """Test that a new wire is added if there are multiple mid-circuit measurements
```

Comment:
Should there also be a test for adding multiple wires because of overlapping measurements?

Reply:
Isn't that what this test does? Or do you mean if there are three or more measurements on the same wire?